tasks/podman.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Install podman package.
+  package:
+    name: podman
+    state: present
+
+- name: Check homedir of retrace user.
+  shell: grep retrace /etc/passwd | cut -d ':' -f6
+  register: retrace_home
+
+- name: Set home directory for retrace user.
```

Comment:
Is there a point in checking here? You could also use the `user` module for this.

Reply:
Willy do.